### PR TITLE
[#270] Fixed S3Hook to load/write logs in S3 properly

### PR DESCRIPTION
**Pull request by aliaksandr-d**
Fixed S3Hook to load/write logs properly to/from S3. #270 
Previously it failed after Kube2IAM implementation. Not it works fine.

Notice: **airflow_s3_log_url** profile property format has been changed.

### CODE REVIEW

**Comment by aliaksandr-d** [comment]
Test this please

File: deploy/profiles/legion-ci.epm.kharlamov.biz.yml [hunk 1]
```diff
@@ -75,7 +75,7 @@ storageclass: efs
 airflow_dags_dir: '/airflow-dags'
 airflow_dags_pvc: legion-airflow-dags
 airflow_s3_bucket_name: 'epm-legion-data-ci'
-airflow_s3_log_url: "s3://{{ airflow_s3_bucket_name }}/{{ enclave }}/airflow-logs/"
+airflow_s3_log_url: "s3://airflow-logs/"
```
Comment (alexander-semenets) [inline deploy/profiles/legion-ci.epm.kharlamov.biz.yml]: Looks like all enclaves at all clusters will log to the same location? Shouldn't there be  a separate log storage per cluster/enclave?

Comment (aliaksandr-d) [reply deploy/profiles/legion-ci.epm.kharlamov.biz.yml]: Airflow will write log files to same directory:
*s3://{{ airflow_s3_bucket_name }}/{{ enclave }}/airflow-logs/*
But it's configured in different way:
1. In airflow configuration: *s3_bucket_path: "{{ enclave }}/"*
2. Bucket name is specified in Airflow connection in secret file.

**Comment by aliaksandr-d** [comment]
Test this please

File: deploy/profiles/legion-ci.epm.kharlamov.biz.yml (hunk 1 above)
Comment (alexander-semenets) [reply deploy/profiles/legion-ci.epm.kharlamov.biz.yml]: worth renaming variable so

**Comment by aliaksandr-d** [comment]
ok to test

**Comment by aliaksandr-d** [comment]
ok to test

**Comment by aliaksandr-d** [comment]
ok to test

**Comment by aliaksandr-d** [comment]
ok to test

**Comment by alexander-semenets** [comment]
ok to test

**Comment by alexander-semenets** [comment]
test this please

**Pull request merged**